known-shortcuts/zoom.ts

```diff
@@ -1,3 +1,5 @@
+import { AppParams } from "~/types";
+
 const app = {
```

Comment:
:bug: **Bug**
The `app` object is being cast to `AppParams` using `as`. It's generally better to use TypeScript's type assertion only when necessary. Consider defining the `app` object with the `AppParams` type directly to ensure type safety. Additionally, there is no verification in the diff to ensure that the `app` object actually conforms to the `AppParams` type. This could lead to runtime errors if the `app` object does not match the expected structure of `AppParams`. Consider using TypeScript's type checking to ensure the `app` object is correctly structured according to `AppParams`.

```suggestion
const app: AppParams = {
```